app/views/volunteers/show.html.slim

```diff
@@ -2,6 +2,11 @@
   .col-xs-12
     h1= @volunteer.contact.full_name
 
+- if Volunteer.seeking_clients.include?(@volunteer)
```

Comment:
BTW the `state` attributes in `Volunteer` and `Client` could be simplified by using [AR enums](http://www.justinweiss.com/articles/creating-easy-readable-attributes-with-activerecord-enums/), but that would also involve switching from strings to integers. Maybe something for a Kata session some day :)